sourcegraph/sourcegraph
# [Search Jobs]: Connect view logs and download results search job UI

This PR just connects download result and view logs buttons to the search job API 

## Test plan
- Check that you can download results for search jobs that have download links (all successful search jobs)
- Check that you can download the logs file
